source/develop/release-management/features/storage/remove-snapshot.html.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Remove Snapshot
+category: feature
+authors: ahino
+wiki_category: Feature|RemoveSnapshot
+wiki_title: Features/Remove Snapshot
+wiki_revision_count: 1
+wiki_last_updated: 2016-09-27
+feature_name: Remove Snapshot
+feature_modules: engine/vdsm
+feature_status: Released
+---
+
+# Remove Snapshot
+
+## Summary
+
+Use qemu-img commit instead of qemu-img rebase when deleting snapshots while the VM is down.
+
+## Owner
+
+*   Name: Ala Hino
+*   Email: <ahino@redhat.com>
+
+## Terminology
+
+This feature page is about removing snapshots when the VM is down. As today we use 'Live Merge' when removing snapshots while the VM is running, I will use 'Cold Merge' when referring to removing snapshots while the VM is down.
```

Comment:
Who is "we"  and "I"?
Maybe something like:
"As today the 'Live Merge' term is used [.....], 'Cold Merge' will be used when [....]